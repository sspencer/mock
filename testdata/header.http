### Get auth loaf
GET /bread/1
Accept-Ranges: bytes
Date: Fri, 07 Oct 2022 06:37:46 GMT
