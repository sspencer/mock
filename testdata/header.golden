Get auth loaf: GET /bread/1 delay=0s status=200 header=89 body=0